automation/Dockerfile

```diff
@@ -3,37 +3,22 @@
 # It enables systemd to be operational.
 FROM centos:7.5.1804
 ENV container docker
-RUN (cd /lib/systemd/system/sysinit.target.wants/; for i in *; do [ $i == \
-     systemd-tmpfiles-setup.service ] || rm -f $i; done); \
-    rm -f /lib/systemd/system/multi-user.target.wants/*;\
-    rm -f /etc/systemd/system/*.wants/*;\
-    rm -f /lib/systemd/system/local-fs.target.wants/*; \
-    rm -f /lib/systemd/system/sockets.target.wants/*udev*; \
-    rm -f /lib/systemd/system/sockets.target.wants/*initctl*; \
-    rm -f /lib/systemd/system/basic.target.wants/*;\
-    rm -f /lib/systemd/system/anaconda.target.wants/*;
+COPY docker_enable_systemd.sh docker_sys_config.sh ./
```

Comment:
This duplication needs to be resolved at an image level: Creating a layered Dockerfile.
One image for CentOS + systemd and the nmstate dev one which depends on it.

I planned to do this as part of https://nmstate.atlassian.net/browse/NMSTATE-76 but I never got to it. If you prefer to do it now, please do but note that docker-hub needs to be updated.

Reply:
After seeing the intro of the Fedora Dockerfile, I think I understand the usefulness of the scripts. It is unfortunate that docker does not allow use to compose images and share this way the code. Using scripts for this is ugly IMO.

Anyway, please include the reasoning source in the commit message if this is indeed the case.